### PR TITLE
Upgrade Bazel to version 0.11.0

### PR DESCRIPTION
**Pull request by rlenglet**
Upgrade Bazel to allow compiling the Istio Envoy filters (cilium/cilium#3035).
Corresponding Gingko VM upgrade: https://github.com/cilium/packer-ci-build/pull/34

Signed-off-by: Romain Lenglet <romain@covalent.io>

### CODE REVIEW

File: envoy/BAZEL_VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.10.1
\ No newline at end of file
+0.11.0
```
Comment (aanm) [inline envoy/BAZEL_VERSION]: I think you need to remove the trailing `\n` (see old file)

**Review by aanm — CHANGES_REQUESTED** [review]

File: envoy/BAZEL_VERSION (hunk 1 above)
Comment (rlenglet) [reply envoy/BAZEL_VERSION]: Good catch. Fixed.

**Comment by rlenglet** [comment]
test-me-please

**Review by manalibhutiyani — APPROVED** [review]

**Review by jrajahalme — CHANGES_REQUESTED** [review]
IMO bazel version upgrades should only be done on the PRs that bump the upstream Envoy dependency. New bazel versions may not always be backwards compatible, i.e, we should use the bazel version that was used to build the upstream Envoy dependency.

Bazel version change may also have an effect to existing bazel caches, and it's not worth it to rebuild the builder container and CI box images separately from the upstream Envoy changes, which will require them as well.

**Comment by rlenglet** [comment]
I agree with you that I could wait before merging those changes in.
However, I have a dependency problem. My PR that builds the Istio filters can’t pass the tests until Basel is upgraded in packer-ci-build.
It’s affecting my development workflow. I can’t test my changes in the CI right now.

**Comment by rlenglet** [comment]
And I guess it’s ok to upgrade in dev Vagrant VM in the Istio filter PR. I’ll do that.

**Comment by rlenglet** [comment]
Abandoning this change. Integrated into https://github.com/cilium/cilium/pull/3035.

**Review by aanm — APPROVED** [review]

**Pull request closed without merging**